src/main/java/frc/robot/Ports.java

```diff
@@ -11,7 +11,25 @@
     }
 
     public static class Controls {
+        public static final int JOYSTICK = 0;
         public static final int XBOX = 0;
     }
 
-}
+    /*
+     left motor's data, right motor's data.
+     */
+    public static class Climber {
+        public static final int MAIN_MOTOR = 0;
+        public static final int AUX = 0;
```

Comment:
Be consistent – either call them `MAIN_MOTOR` and `AUX_MOTOR` or `MAIN` and `AUX`.
Anyway, "follower" is better. That's the term CTRE uses and "auxiliary" means other things in CTRE.